src/modules/universal-discovery/components/finder/finder.tree.leaf.component.js

```diff
@@ -131,7 +131,7 @@ export default class FinderTreeLeafComponent extends Component {
         return (
             <div {...attrs}>
                 {this.renderIcon()}
-                {location.ContentInfo.Content.Name}
+                <span className="c-finder-tree-leaf__content-name">{location.ContentInfo.Content.Name}</span>
```

Comment:
Note: ellipsis stopped working after adding `display: flex` on `c-finder-tree-leaf`, so I wrapped content name in `span`.